src/basic/btrfs-util.c

```diff
@@ -1881,7 +1877,6 @@ int btrfs_subvol_auto_qgroup_fd(int fd, uint64_t subvol_id, bool insert_intermed
                 uint64_t lowest = 256, new_qgroupid;
                 bool created = false;
                 int i;
-
```

Comment:
This `i` can be also dropped.

Please do not drop this blank line.

Reply:
Why do not you declare `i` in each loop?

Reply:
that can be done yeah